### PR TITLE
Pre-organize MSWC FSCIL example code

### PR DESCRIPTION
**Pull request by V0XNIHILI**
Creating separate branch to avoid conflicts while working. Will merge it later back to the original PR branch.

### CODE REVIEW

File: neurobench/examples/mswc_fscil/mswc_fscil_proto.py [hunk 1]
```diff
@@ -1,59 +1,60 @@
...
+from neurobench.examples.mswc_fscil.sparchSNNs import RadLIFLayer
 
 from neurobench.benchmarks import Benchmark
-from neurobench.preprocessing import MFCCProcessor
-from neurobench.preprocessing import S2SProcessor
-
-from cl_utils import *
+from neurobench.preprocessing import MFCCProcessor, S2SProcessor
 
+from weight_consolidation_utils import examples_per_class
```
Comment (Maxtimer97) [inline neurobench/examples/mswc_fscil/mswc_fscil_proto.py:27]: Not needed actually

File: neurobench/examples/mswc_fscil/mswc_fscil_proto.py [hunk 2]
```diff
@@ -348,26 +351,23 @@ def pre_train(model):
...
-            benchmark_all_test = Benchmark(eval_model, metric_list=[static_metrics, data_metrics], dataloader=test_loader, 
+            benchmark_all_test = Benchmark(eval_model, metric_list=[static_metrics, workload_metrics], dataloader=test_loader, 
                                 preprocessors=[to_device, encode, squeeze], postprocessors=[])
 
             benchmark_new_classes = Benchmark(eval_model, metric_list=[[],["classification_accuracy"]], dataloader=test_loader,
                                 preprocessors=[to_device, encode, squeeze], postprocessors=[])
             
-
             cur_class = support[0][1].tolist()
             eval_model.net.cur_j = examples_per_class(cur_class, 200, 5)
```
Comment (Maxtimer97) [inline neurobench/examples/mswc_fscil/mswc_fscil_proto.py:366]: Can be deleted. cur_j is not used

File: neurobench/examples/mswc_fscil/mswc_fscil_proto.py (hunk 1 above)
Comment (Maxtimer97) [inline neurobench/examples/mswc_fscil/mswc_fscil_proto.py:27]: Not needed actually

**Review by Maxtimer97 — COMMENTED** [review]
Sounds good. 
Not sure if we want to keep the weight consolidation method at all and maybe we also want to remove parsers in the base code but I guess it's OK as is.

We mostly need to finish preparing a nice tutorial now I guess :)

And I can do a final cleaning with more commenting and maybe better choice of prototype method variables.

File: neurobench/examples/mswc_fscil/mswc_fscil_proto.py (hunk 2 above)
Comment (Maxtimer97) [reply neurobench/examples/mswc_fscil/mswc_fscil_proto.py:366]: About `eval_model.net.cur_j = examples_per_class(cur_class, 200, 5)`

**Comment by V0XNIHILI** [comment]
@jasonlyik we are sure we are keeping the weight consolidation method right?

**Comment by jasonlyik** [comment]
@V0XNIHILI 
I'll leave it up to you, since we do have code written if we can clean it up for release without too much hassle then we don't lose anything by including it. But it isn't reported in the paper so there is no particular need for it to be included

File: neurobench/examples/mswc_fscil/mswc_fscil_proto.py (hunk 2 above)
Comment (V0XNIHILI) [reply neurobench/examples/mswc_fscil/mswc_fscil_proto.py]: Resolved by 37346879901a8255ef2af844f2b42a1e5347c58b

File: neurobench/examples/mswc_fscil/mswc_fscil_proto.py (hunk 1 above)
Comment (V0XNIHILI) [reply neurobench/examples/mswc_fscil/mswc_fscil_proto.py]: Resolved by #b7dfcfa4be8984550be14da395c6a068933587b9.

**Pull request merged**